server/lib/project/contracts/ProjectManager.sol

```diff
@@ -98,6 +98,18 @@ contract ProjectManager is ErrorCodes, Util, ProjectState, ProjectEvent, BidStat
     return bid.settle(supplierAddress);
   }
 
+  function rejectProject(string name, address bidAddress) returns (ErrorCodes) {
+    // validity
+    if (!exists(name)) return (ErrorCodes.NOT_FOUND);
+    // set project state
+    address projectAddress = getProject(name);
+    var (errorCode, state) = handleEvent(projectAddress, ProjectEvent.RECEIVE);
+    if (errorCode != ErrorCodes.SUCCESS) return errorCode;
+    // reject
+    Bid bid = Bid(bidAddress);
+    return bid.reject(msg.sender);
```

Comment:
@liorsaar1 can I assume that if the buyer is logged in that `msg.sender` will be their address?